common/containers/Tabs/SendTransaction/index.tsx

```diff
@@ -215,7 +215,7 @@ export class SendTransaction extends React.Component<Props, State> {
   }
 
   public handleWalletStateOnUpdate(prevProps) {
-    if (this.props.wallet !== prevProps.wallet) {
+    if (this.props.wallet !== prevProps.wallet && prevProps.wallet !== null) {
```

Comment:
Should probably change to `... && prevProps.wallet` to check for any falsey value

Reply:
You mean something more like this: ` ... && !!prevProps.wallet`?

Reply:
Both would work